sky/clouds/service_catalog/images/aws_image_gen.py

```diff
@@ -0,0 +1,82 @@
+"""
+Run `python aws_image_gen.py` to copy the AMI in the source region to all regions and make them public.
+Will also generate a `images.csv` or append to an existing one; then we can refresh the `images.csv` inSkyPilot Catalog.
+"""
```

Comment:
style nit: docstr should have the first line that is a short description and the following details can be lone

```python
"""Copy SkyPilot AMI to multiple regions, make them public, and generate images.csv

This script will copy the source image to all regions, and generate the images.csv for future SkyPilot usage. If `images.csv` exists, it will append the new images to the file. 

Example Usage:
  python aws_image_gen.py  --source-image-id ${IMAGE_ID} --base-image-id ${BASE_IMAGE_ID}
"""
```